layouts/faq/list.html

```diff
@@ -1,16 +1,328 @@
 {{ define "main"}}
 
 <section class="features section padding-bottom footer-page">
-    <div class="container page-background" >
-        <div class="row">
-            <div class="col-md-12 col-sm-12 col-xs-12">
-                <h1 class="title">FAQ</h1>
-                <div class="content">
-                    {{.Content}}
+    <div class="container page-background" style="text-align: center; margin-top: 80px;" >
+        <div class="row text-center" style="text-align: center;">
+            <div class="col-*-12"style="margin: auto;">
+          <div class="col-xl col-md col-sm-auto col-xs-auto faq activefaq" style="display: inline-block;" id="button1" onclick="troubleshooting()" >
+              <h2>Troubleshooting</h2>
+
+          </div>
+          <div class="col-xl col-md col-sm-auto col-xs-auto faq" style="display: inline-block;" id="button2" onclick="education()">
+            <h2>Education</h2>
+          </div>
+          <div class="col-xl col-md col-sm-auto col-xs-auto faq" style="display: inline-block;" id="button3" onclick="aboutorange()" >
+            <h2>About Orange</h2>
+          </div>
+          <!-- <div class="col-xl col-md col-sm-auto col-xs-auto  faq" style="display: inline-block;" id="button4" onclick="aboutus()" >
```

Comment:
Should be removed before merge, along with the function and references to the button in the other categories' corresponding functions.